fuzzers/int_maketodo.py

```diff
@@ -86,12 +96,58 @@ def maketodo(
             include = re.match(exclude_re, line) is None
 
         if include:
-            print(line)
+            filtered_todos.add(line)
         else:
             drops += 1
         lines += 1
     verbose and print('Print %u entries w/ %u drops' % (lines, drops))
 
+    if balance_wire_re is not None:
```

Comment:
Please add a comment which describes what balancing is, why it is needed and the theory of how it works.

Reply:
You wrapping of these comments is very strange. Can you rewrap it?

Reply:
Should be better now?